distage/distage-framework-docker/src/test/scala/izumi/distage/testkit/docker/DistageTestDockerBIO2.scala

```diff
@@ -0,0 +1,44 @@
+package izumi.distage.testkit.docker
+
+import distage.DIKey
+import izumi.distage.docker.examples.DynamoDocker
+import izumi.distage.testkit.TestConfig
+import izumi.distage.testkit.docker.fixtures.PgSvcExample
+import izumi.distage.testkit.scalatest.DistageBIOSpecScalatest
+import izumi.fundamentals.platform.properties.EnvVarsCI
+import izumi.logstage.api.Log
+import zio.IO
+
+// this tests needed to check mutex for reusable containers during parallel test runs
+final class DistageTestDockerBIO2 extends DistageBIOSpecScalatest[IO] {
```

Comment:
@Caparow Could you make top level classes with this test like 10-15 times with `moduleOverrides = new ModuleDef { make[UUID].fromValue(randomUUID()) }` to guarantee new test envs - to be able to check that the dockers do not duplicate by manual testing

Reply:
done